src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/FilterVariantTranches.java

```diff
@@ -26,7 +26,13 @@
 import picard.cmdline.programgroups.VariantFilteringProgramGroup;
 
 /**
- * Apply tranche filtering to VCF based on scores from an annotation in the INFO field.
+ * Apply tranche filtering to VCF based on scores from an annotation in the INFO field.  Tranches are specified in
+ * percent sensitivity to the variants in the resource files.  For example, if you specify INDEL tranches 98.0 and 99.0
+ * using the CNN_2D score the filtered VCF will contain 2 filter tranches for INDELS: CNN_2D_INDEL_Tranche_98.00_99.00
+ * and CNN_2D_INDEL_Tranche_99.00_100.00. We expect CNN_2D_INDEL_Tranche_99.00_100.00 to be more sensitive but less precise
```

Comment:
You don't have a 100+ like VQSR?  (where 100+ is for stuff that scores worse than the worst of the resource variants)

Also, you should be explicit that anything scoring better than the indel tranche 98 is PASS.

Reply:
Added note that all other variants pass.  We put the worse than the worst variants into the _100.00 tranche too, which I guess should be called the 100.00+.  Do you think we should we add the plus sign?